client/layout/index.js

```diff
@@ -76,9 +76,13 @@ class Layout extends Component {
 
 	render() {
 		const { isEmbedded, ...restProps } = this.props;
+		const { breadcrumbs } = this.props.page;
 		return (
 			<div className="woocommerce-layout">
-				<Slot name="header" />
+				<Header
+					sections={ ( isFunction( breadcrumbs ) ? breadcrumbs( this.props ) : breadcrumbs ) || [] }
```

Comment:
Considering pages can be added by 3PD, it would be useful to elaborate `Layout.propTypes` for `page` and make breadcrumbs required. Then, this ternary + `||` expression can be simplified and we get better checks against misuse.

Reply:
Sure, done in https://github.com/woocommerce/woocommerce-admin/pull/2491/commits/b4e3bc0323936522a7eb83d85f95237f99e09b59 👍